### PR TITLE
made style changes

### PR DESCRIPTION
**Pull request by cschmieley71**

### CODE REVIEW

File: verksamhetsOmraden/verksamhetsOmraden.html [hunk 1]
```diff
@@ -3,35 +3,39 @@
...
+
     
 </head>
 <body>
-    <h2>
-    Konst- och kulturutbildningar
-    </h2>
-
-    <p>
-        Inom konst- och kulturutbildningar finns det flera olika inriktningar
```
Comment (Aiiion) [inline verksamhetsOmraden/verksamhetsOmraden.html:15]: Should this really be removed?

File: verksamhetsOmraden/verksamhetsOmraden.html [hunk 2]
```diff
@@ -3,35 +3,39 @@
...
-    </ul>
+    </p>
 
-    <footer>
+
+    <footer style="color: grey;">
         E-post
         Allmänna frågor/synpunkter: info@myh.se
         Registrator: registrator@myh.se
     </footer>
```
Comment (Aiiion) [inline verksamhetsOmraden/verksamhetsOmraden.html:36]: Please replace with actuall footer from templates/footer

File: verksamhetsOmraden/verksamhetsOmraden.html [hunk 3]
```diff
@@ -3,35 +3,39 @@
...
-    </p>
-    <h2>
-        Myndighetens uppgifter för konst- och kulturutbildningar
-    </h2>
-    <ul>
-        <li> Fatta beslut om vilka utbildningsanordnare som ska beviljas statligt stöd (statsbidrag och/eller
-        studiestödsberättigande) </li>
+    
+    
+    <ul style="color: grey;">
```
Comment (Aiiion) [inline verksamhetsOmraden/verksamhetsOmraden.html:24]: lets try not putting css in the html file, it gets messy fast

**Review by Aiiion — CHANGES_REQUESTED** [review]
Go though this one more time and add the header while you are at it.
Also add grid to the site and the finished styling for footer and header.

File: verksamhetsOmraden/verksamhetsOmraden.html (hunk 2 above)
Comment (cschmieley71) [reply verksamhetsOmraden/verksamhetsOmraden.html:36]: I have replaced the previous footer with the footer in the template/footer

File: verksamhetsOmraden/verksamhetsOmraden.html (hunk 3 above)
Comment (cschmieley71) [reply verksamhetsOmraden/verksamhetsOmraden.html:24]: Removed

**Pull request closed without merging**